biofefi/services/fi.py

```diff
@@ -78,3 +92,66 @@ def calculate_local_shap_values(
 
     # Return the DataFrame
     return shap_df, shap_values
+
+
+def load_fi_options(path: Path) -> dict:
+    """Load feature importance options.
+
+    Args:
+        path (Path): The path to the feature importance options file.
+
+    Returns:
+        dict: The feature importance options.
+    """
+
+    try:
+        with open(path, "r") as file:
+            fi_options = json.load(file)
+    except FileNotFoundError:
+        fi_options = None
+
+    return fi_options
```

Comment:
Since this is about getting some configuration from a file, can you move it to `services.configuration` please. Also can you make this function return a `FeatureImportanceOptions` object please.